platform/core.startup/build.xml

```diff
@@ -21,4 +21,14 @@
 -->
 <project name="platform/core.startup" default="build" basedir=".">
     <import file="../../nbbuild/templates/projectized.xml"/>
+    <target name="build-init" depends="projectized.build-init">
+      <taskdef name="splash-screen" classname="org.netbeans.nbbuild.assets.GenerateSplashScreenTask" classpath="${nbantext.jar}" />
+      <echo message="Generating splash screens with build number ${buildnum}" />
+      <splash-screen 
+        font='../../nbbuild/external/montserrat-medium.ttf' 
+        output='src/org/netbeans/core/startup/splash.gif'>${buildnum}</splash-screen>
```

Comment:
I wonder - is there a strong reason to generate the files into the source directories? Wouldn't it work to generate them to '${build.classes.dir}' (may require <mkdir dir="${build.classes.dir}"> as well)? That would keep the sources clean, and would avoid the need to tweak .gitignore.